### DISCUSSION TITLE
Extend Metadata Features

### DISCUSSION THREAD

**Discussion by cyco77** [Power-Maverick/DataverseDevTools-VSCode#116] [Ideas]
I would like to extend the metadata explorer. 

My idea would be to create a new entry in "Power Platform Toolbox". This view can show all entites (simular to existing treeview, but with more details). 

Next step would be drill down to entity detail view (existing one). Here I would also show relationships, keys .... can be extended step by step.

If it's ok for you, I would start realizing this feature.

**Comment by Power-Maverick** [comment]
Are you planning on adding a web view or just enabling the tree view. If it's the later then please expand the existing entities tree view.

Power Platform Toolbox is for tools only which may use CLI or web view.

**Comment by cyco77** [comment]
I would prefer a webview. there I can display more details as in the tree.
And on the other hand it's more extensible as the tree... as I wrote in my initial post, drill down to the existing detailspage could be possible.

**Comment by cyco77** [comment]
I started development on this feature... right now I placed a new button in the treeview:
![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/1198698/25b306dd-1630-4350-8b58-a46281181741)

right now it shows this new webview:
![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/1198698/e068874d-1936-48fb-a904-bf04fe445c30)

I'm still at the beginning, but I like the way to open this view...

**Comment by Power-Maverick** [comment]
Released in v2.1.4
